requirements.txt

```diff
@@ -20,6 +23,7 @@ Pillow==8.1.0
 pycparser==2.20
 PyJWT==1.7.1
 PyMySQL==0.10.1
+python-magic==0.4.18
```

Comment:
Muss das auch noch in die `settings.py`?

Reply:
Ich denke nicht. Das ist ja keine Django-App.

Reply:
`magic` ist die django-app, oder nicht? Ich bin mir grad nicht sicher, die anderen haben wir auch alle drin...

Reply:
Okay ich glaub, ich habs verstanden, ich räum die anderen mal auf, ich glaub wir haben da ein bisschen was durcheinander...